documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc

```diff
@@ -50,6 +69,9 @@ run-hub-lvmo-task:
     			--use-param-defaults hub-deploy-lvmo \
     			--showlog
 ```
+    NOTE:: You'll need to make sure that the HUB's master nodes have one available block device to form the required volume group. The total size of these block devices should be no less than 900GB.
```

Comment:
I would rephrase this as: 

Please, make sure to have an available, default, storageclass configured in the hub cluster for the PVC to be properly bound.

Reply:
This note is for the automated PV creation - we need the block devices, right?

Reply:
ah, good point. The diff confused me and I thought it was a generic note. It's correct, then.